Backend/databaseInteraction/databaseInterface.js

```diff
@@ -7,16 +7,14 @@ const model = require('../db/db-init.js')
  * @returns the user object, if found, otherwise false
  */
 async function doCredentialsMatch(email, password) {
-  const isSetup = await model.isSetup();
-  console.log('x', isSetup)
-  if (!isSetup) return false;
-  const employees = await model.Employee.findAll({
+  const employees = await models.Employee.findAll({
+    attributes: { exclude: ['password','email'] },
     where: {
       email: email,
       password: password,
     }
```

Comment:
If you just want the raw JSON values, use the `'raw': true` flag here. That's faster and doesn't require to use `.dataValues ` which can be confusing